### ISSUE TITLE
Reloading development server causes undefined method `swagger_controller'

### ISSUE THREAD

**Issue by rymndhng** [richhollis/swagger-docs#27]
Trying out swagger-docs adn it's pretty sweet.

However -- on a development server where I'm allowing automatic reload -- I end up with `undefined method`swagger_controller'` unless I restart the development server.

Am I configuring something wrong for this to happen?

**Comment by robdbirch** [comment]
I see this was resolved in #29. But, I'm getting this issue too, added @wndxiori solution and it works after development controller changes.

**Comment by benmort** [comment]
ldnunes fixed for me
tweaked slightly to work in my case

```
class Api::BaseController < ActionController::Base
  class << self
    Swagger::Docs::Generator::set_real_methods

    def inherited(subclass)
      super
      subclass.class_eval do
        setup_basic_api_documentation
      end
    end

    private
    def setup_basic_api_documentation
      [:index, :show, :create, :update, :delete].each do |api_action|
        swagger_api api_action do
          param :header, 'Authentication-Token', :string, :required, 'Authentication token'
        end
      end
    end
  end
end
```

```
module Api
  module V1
    class DistanceController < Api::BaseController
    end
  end
end
```
